src/connector/src/parser/util.rs

```diff
@@ -84,7 +86,15 @@ pub(super) fn at_least_one_ok(mut results: Vec<Result<()>>) -> Result<()> {
 
     if let Some(first_ok_index) = first_ok_index {
         if !err_message.is_empty() {
-            tracing::error!("failed to parse some columns: {}", err_message)
+            static LOG_SUPPERSSER: LazyLock<LogSuppresser> =
```

Comment:
This function seems to be dead.

Reply:
Indeed. Let me remove it. PTAL @tabVersion